tox.ini

```diff
@@ -371,10 +371,14 @@ basepython = {[default]basepython}
 
 recreate = true
 
-deps =
+deps = pipdeptree
 
 commands =
-    pip freeze --exclude={env:PY_MODULE}
+    python -c 'print()'
+    pip freeze --exclude="{env:PY_MODULE}"
+
+    python -c 'print()'
+    pipdeptree
```

Comment:
This prints out the pip freeze output, which I use to compare against what is in `setup.py` occasionally, and it now adds a dependency tree which is handy for seeing why certain things are showing up.

Reply:
I am happy to see pipdeptree here :)